packages/astro-embed-link-preview/LinkPreview.astro

```diff
@@ -35,20 +35,18 @@ const domain = meta?.url ? new URL(meta.url).hostname.replace('www.', '') : '';
 				</header>
 				<small class="link-preview__description">{meta.description}</small>
 			</div>
-			{meta.video && meta.videoType ? (
+			{!hideMedia && meta.video && meta.videoType ? (
```

Comment:
Wonder if instead of `!hideMedia` here and in in the image check, we can simplify that to:
```suggestion
			{hideMedia ? null : meta.video && meta.videoType ? (
```